docs/docusaurus.config.js

```diff
@@ -114,6 +108,12 @@ const config = {
         defaultMode: 'light',
       },
       metadata: [{name: 'akuity, argoproj, argo cd, argo workflows, argo events, argo rollouts, kubernetes, gitops, devops', content: 'akuity, documentation, developer documentation'}],
+      algolia: {
+        appId: '3SQ7LK6WD9',
+        apiKey: '5627b8c2efd5b28a5b70c6660cb2b0f3',
```

Comment:
Few questions here:

1. Is this safe to commit in a public repo? I'm assuming it may be. That's common with Google Analytics integration and I guess this is similar?

2. How does this work with the multiple versions of the docs site we host simultaneously (one for each minor revision of Kargo)? They can all use the same ID? Or no? Are they all using the same search index? Or are they each indexed separately?

Reply:
1. Yes, this API key is [safe to commit](https://support.algolia.com/hc/en-us/articles/18966776061329-Can-the-search-API-key-be-public).
2. The search results on the version subdomains will, at this moment, return results for the latest version. I have yet to figure out how to avoid this. But given this won't be a problem until we release 1.6.0, I would not like to make this a blocker given the value provided as soon as we merge this.

Reply:
I agree it should not block this -- even if we didn't figure it out for a while. Doing a search in an older version of the docs and getting results from the newest version is a little weird, but it's not anything I'm going to lose sleep over.